catroid/src/main/java/org/catrobat/catroid/stage/StageActivity.java

```diff
@@ -438,4 +458,50 @@ public void run() {
 	public static int getAndIncrementNumberOfClonedSprites() {
 		return ++numberOfSpritesCloned;
 	}
+
+
+
+	public synchronized void queueIntent(IntentListener asker) {
+		int newIdentId;
+		do {
+			newIdentId = StageActivity.randomGenerator.nextInt(100000);
```

Comment:
is there a reason for the bound of 100000? why not full integer range? is 0 a valid number here?